airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -64,13 +67,12 @@
 from airflow.utils.db import get_query_count
 from airflow.utils.state import TaskInstanceState
 
-task_instances_router = AirflowRouter(
-    tags=["Task Instance"], prefix="/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances"
-)
+task_instances_router = AirflowRouter(tags=["Task Instance"], prefix="/dags/{dag_id}")
+task_instances_prefix = "/dagRuns/{dag_run_id}/taskInstances"
```

Comment:
should we make `task_instances_prefix` all capitals to indicate constant?

Reply:
We can make it all capitals but if we plan to do so, I suppose we'll need to follow the same for all router variables too, for consistency.

E.g. we'll need to update `task_instances_router` to `TASK_INSTANCE_ROUTER`, etc.